let items;
try {
    // bu kod hata items olmadığından hata vercek
    // try içinde kullanmıyorsak aşşağıdaki
    // geçiş yazısınıda göremeyiz
    items.forEach(items => {
        
    });
    
} catch (error) {
    console.log(error)
}


console.log('geçiş')